app/controllers/participants/validations_controller.rb

```diff
@@ -202,7 +191,12 @@ def step_valid?
     def store_form_and_redirect_to_step(step)
       @participant_validation_form.step = step
       session[:participant_validation] = @participant_validation_form.attributes
-      redirect_to send("participants_validation_#{step}_path")
+
+      if request.get? && change?
+        redirect_to send("participants_validation_#{step}_path", change: true)
+      else
+        redirect_to send("participants_validation_#{step}_path")
+      end
```

Comment:
Why does it need to be a `request.get?`?

Reply:
Because of the way the logic works when handling requests, the change flag is just to indicate that the request is to change some data and that we should reattempt validation when the form is submitted. If the flag is also set on a put it loops on the same page. It's only needed now that the TRN is split out on its own page at the beginning.